app/frontend/components/uppy-file-upload.js

```diff
@@ -4,27 +4,40 @@ import DropTarget from "@uppy/drop-target";
 import Webcam from "@uppy/webcam";
 import ActiveStorageUpload from "./uppy/ActiveStorageUpload";
 
-
 const uppyFileUpload = (container, options = {}) => {
   const uppy = new Uppy({
     id: container.id,
+    locale: {
+      strings: {
+        youCanOnlyUploadX: {
+          0: "You can only upload %{smart_count} file at a time",
```

Comment:
Is this correct? It looks like it's for the singular one

Reply:
Yeah, I can see how this is confusing. This is more of a boolean "does this have more than 1" style definition, rather than an actual count definition: https://github.com/transloadit/uppy/blob/main/packages/%40uppy/utils/src/Translator.js#L83